### PR TITLE
Spanish translations setup draw

### PR DESCRIPTION
**Pull request by lauradowell**
Added a Spanish translated version of the setup() function. I also made sure that "Structure" module and submodule is referred to as "Structure" In other Spanish translations on the website

### CODE REVIEW

File: src/content/reference/es/p5/setup.mdx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: setup
+module: Structure
+submodule: Structure
+file: src/core/main.js
+description: >
+ <p>La función <a href="/reference/p5/setup">setup()</a> es llamada una vez al comienzo del programa.  Se utiliza para
```
Comment (Juan31416) [inline src/content/reference/es/p5/setup.mdx]: ### Original:
```html
<p>La función <a href="/reference/p5/setup">setup()</a> es llamada una vez al comienzo del programa. Se utiliza para
```
### Improved NIT:
```html
 <p>La función <a href="/reference/p5/setup">setup()</a> es llamada <b>una única vez</b> al comienzo del programa. Se utiliza para
```

File: src/content/reference/es/p5/setup.mdx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+definir propiedades iniciales del entorno, como el tamaño de la pantalla y el color del fondo
```
Comment (Juan31416) [inline src/content/reference/es/p5/setup.mdx]: ### Original:
```html
definir propiedades iniciales del entorno, como el tamaño de la pantalla y el color del fondo
```
### Improved NIT:
```html
 <b>configurar el entorno inicial</b>, como el tamaño de la pantalla, el color del fondo, 
```

File: src/content/reference/es/p5/setup.mdx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+y para cargar archivos multimedia como imágenes y fuentes en la iniciación del programa.
```
Comment (Juan31416) [inline src/content/reference/es/p5/setup.mdx]: ### Original:
```html
y para cargar archivos multimedia como imágenes y fuentes en la iniciación del programa.
```
### Improved NIT:
```html
y para cargar archivos multimedia, imágenes y fuentes, al inicializar el programa.
```

File: src/content/reference/es/p5/setup.mdx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+Solo puede haber una función <a href="/reference/p5/setup">setup()</a> por
```
Comment (Juan31416) [inline src/content/reference/es/p5/setup.mdx]: ### Original:
```html
Solo puede haber una función <a href="/reference/p5/setup">setup()</a> por
programa y no debería volverse a llamar después de su ejecución inicial.</p>
```
### Improved NIT:
```html
<p>Solo puede haber una función <a href="/reference/p5/setup">setup()</a> por
programa y no debería volverse a llamar después de su ejecución inicial.</p>
```

Separate last part into a new paragraph.

**Review by Juan31416 — APPROVED** [review]

File: src/content/reference/es/p5/setup.mdx [hunk 5]
```diff
@@ -0,0 +1,41 @@
+---
+title: setup
+module: Structure
+submodule: Structure
+file: src/core/main.js
+description: >
+  <p>La función <a href="/reference/p5/setup">setup()</a> es llamada una vez al comienzo del programa.  Se utiliza para
+  definir propiedades iniciales del entorno, como el tamaño de la pantalla y el color del fondo
+  y para cargar archivos multimedia como imágenes y fuentes en la iniciación del programa.
+  Solo puede haber una función <a href="/reference/p5/setup">setup()</a> por
```
Comment (Juan31416) [inline src/content/reference/es/p5/setup.mdx:10]: ### Original:
```html
Solo puede haber una función <a href="/reference/p5/setup">setup()</a> por
programa y no debería volverse a llamar después de su ejecución inicial.</p>
```
### Improved NIT:
```html
<p>Solo puede haber una función <a href="/reference/p5/setup">setup()</a> por
programa y no debería volverse a llamar después de su ejecución inicial.</p>
```

Separate last part into a new paragraph.

File: src/content/reference/es/p5/draw.mdx [hunk 6]
```diff
@@ -0,0 +1,84 @@
+---
+title: draw
+module: Structure
+submodule: Structure
+file: src/core/main.js
+description: >
```
Comment (Juan31416) [inline src/content/reference/es/p5/draw.mdx:6]: This one is hard to read if directly translated into Spanish. So, I suggest the following structure: 

```html
    <p>
      La función <code>draw()</code> es la función central de un sketch. Se
      ejecuta de forma continua después de la función
      <a href="/reference/p5/setup">setup()</a>, hasta que el programa se
      detenga o se llame a la función <code>noLoop()</code>. La frecuencia de
      ejecución se establece por defecto en 60 fotogramas por segundo, y podemos
      ajustarlo mediante la función
      <a href="/reference/p5/frameRate">frameRate()</a>.
    </p>

    <p>
      Podemos controlar cuándo y cómo se ejecuta <code>draw()</code> con las
      funciones:
    </p>

    <ul>
      <li>
        <a href="/reference/p5/noLoop">noLoop()</a> - Detiene la ejecución de
        cíclica de <code>draw()</code>.
      </li>
      <li>
        <a href="/reference/p5/loop">loop()</a> - Reanuda la ejecución de
        cíclica <code>draw()</code>.
      </li>
      <li>
        <a href="/reference/p5/redraw">redraw()</a> - Ejecuta
        <code>draw()</code> una sola vez después de haber sido invocada.
      </li>
    </ul>

    <p>
      Nota: si se llama la función <code>noLoop()</code> dentro de <code>setup()</code>,
      <code>draw()</code> se ejecutará una vez más antes de detenerse.
    </p>

    <p>Consideraciones importantes:</p>
    <ul>
      <li>Solo puede haber una función <code>draw()</code> por sketch.</li>
      <li>
        La función <code>draw()</code> es indispensable para generar animaciones
        o procesar eventos como
        <a href="/reference/p5/mousePressed">mousePressed()</a>.
      </li>
      <li>
        Los estilos aplicados, como
        <code>fill()</code> y <code>stroke()</code>, se mantienen entre
        ejecuciones consecutivas de <code>draw()</code>.
      </li>
      <li>
        Al inicio de cada ejecución de <code>draw()</code>, el sistema de
        coordenadas se reinicia. Las transformaciones aplicadas previamente,
        como escalar, rotar y traslación, se eliminan evitando su acumulación.
      </li>
    </ul>
```

**Review by Juan31416 — COMMENTED** [review]
Hi @lauradowell 

Thank you so much for translating the documentation! I appreciate it.

I looked at the Spanish version and made suggestions to improve readability.

Please let me know if there's anything else I can help with.

File: src/content/reference/es/p5/draw.mdx (hunk 6 above)
Comment (lauradowell) [reply src/content/reference/es/p5/draw.mdx:6]: these look good

**Comment by limzykenneth** [comment]
@lauradowell You will still need to pull the changes to this PR and make the changes Juan suggested. Let me know if you need help completing that.

**Comment by lauradowell** [comment]
Hi @limzykenneth I have added @Juan31416 's changes. I had some formatting issues so It took me a few attempts to fix these, but Juan's suggestions are incorporated

**Pull request merged**